README.md

```diff
@@ -1,9 +1,9 @@
 # Operators for Qiskit
 
-###  - [Qiskit developer's exp operator](https://github.com/husky-parul/glowing-quantum/tree/qiskit-dev-op/operators-examples/qiskit-dev-operator)
+###   [Qiskit developer's experience operator](https://github.com/husky-parul/glowing-quantum/tree/qiskit-dev-op/operators-examples/qiskit-dev-operator)
 
-Operator sets up the development environment with an integrated Jupyter Notebook and pre installed dependecy for running quantum circuits on [IBMQ Account](https://quantum-computing.ibm.com/) using [Qiskit](https://qiskit.org/)
+Operator sets up a development environment with an integrated Jupyter Notebook and pre-installed dependecy for running quantum circuits on [IBMQ Account](https://quantum-computing.ibm.com/) using [Qiskit](https://qiskit.org/).
     
 Example notebook implementing Grover's Search Algorithm
```

Comment:
With no link I'd expect this sentence to being with "Includes an example...". Otherwise it reads better if it's a link...